css/styles.css

```diff
@@ -1,715 +1,1549 @@
+  background-color: #000;
+  font-size: 12px;
+  margin: 0 auto;
+}
+
+.footer-container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.footer-top {
+  display: flex;
+  padding: 66px 0 37px 4px;
+}
+
+.footer-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-box:first-child {
+  margin: 0 114px 0 0;
+}
+
+.footer-box:nth-child(2) {
+  margin: 0 72px 0 0;
+}
+
+.footer-box:nth-child(3) {
+  margin: 0 85px 0 0;
+}
+
+.footer-box__text {
+  text-transform: uppercase;
+  font-weight: bold;
+  margin: 0 0 8px 0;
+}
+
+.footer-box:last-child 
+.footer-box__text {
+  margin-bottom: 5px;
+}
+
+.footer-box__link {
+  color: #fff;
+  text-decoration: none;
+  margin: 0 0 5px 0;
+}
+
+.footer-box:last-child 
+.footer-box__link {
+  margin-bottom: 10px;
+}
+
+.footer-box__btn {
+  width: 136px;
+  height: 30px;
+  text-align: center;
+  background-color: #000;
+  color: #fff;
+  border: 2px solid #fff;
+  font-weight: bold;
+  font-size: 12px;
+  margin: 0 0 35px 0;
+}
+
+.footer-box__btn:hover,
+.footer-box__btn:focus {
+  background-color: #fff;
+  color: #000;
+  cursor: pointer;
+  border-color: #000;
+}
+
+.footer-box__btn:active {
+  background-color: red;
+}
+
+.footer-icons__unit {
+  margin: 0 22px 0 0;
+}
+
+.footer-bottom {
+  padding: 0 4px 33px 0;
+}
+
+.footer-bottom__text {
+  text-align: center;
+}
+
+@media screen and (max-width: 1024px) {
+  .footer-top {
+    padding: 67px 0 38px 4px;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .footer-top {
+    padding: 57px 0 57px 9px;
+  }
+
+  .footer-box:first-child {
+    margin: 0 54px 0 0;
+  }
+
+  .footer-box:nth-child(2) {
+    margin: 0 67px 0 0;
+  }
+
+  .footer-box:nth-child(3) {
+    margin: 0 68px 0 0;
+  }
+
+  .footer-box__text {
+    margin: 0 0 5px 0;
+  }
+
+  .footer-box:last-child
+  .footer-box__text {
+    margin-bottom: 3px;
+  }
+
+  .footer-box:last-child
+  .footer-box__link {
+    margin-bottom: 13px;
+    width: 155px;
+    letter-spacing: 1px;
+    line-height: 12px;
+  }
+
+  .footer-box__btn {
+    width: 143px;
+    height: 24px;
+    margin: 0 0 27px 0;
+  }
+
+  .footer-icons__unit {
+    margin: 0 16px 0 0;
+  }
+
+  .footer-bottom {
+    padding: 0 0 33px 0;
+  }
+}
+
+@media screen and (max-width: 585px) {
+  .footer {
+    font-size: 15px;
+    text-align: center;
+  }
+
+  .footer-top {
+    flex-direction: column;
+  }
+
+  .footer-box:first-child,
+  .footer-box:nth-child(2),
+  .footer-box:nth-child(3) {
+    margin: 0 0 41px 0;
+  }
+}
+
+@media screen and (max-width: 375px) {
+  .footer-top {
+    padding: 36px 0 16px 0;
+  }
+
+  .footer-box__text {
+    margin: 0 0 3px 0;
+  }
+
+  .footer-box__link {
+    margin: 0 0 7px 0;
+  }
+
+  .footer-box:first-child {
+    order: -2;
+  }
+
+  .footer-box:nth-child(3) {
+    order: -1;
+  }
+
+  .footer-box:nth-child(3) 
+  .footer-box__text {
+    letter-spacing: 1px;
+  }
+
+  .footer-box:nth-child(3) 
+  .footer-box__link,
+  .footer-box:nth-child(2) 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Itogovya_axion/pullRequest?prid=6118030)